### PR TITLE
Add bash autocompletion for faf

### PR DESCRIPTION
**Pull request by mkutlak**
You won't have to remember all the commands from now on or type --help
to run faf commands.*

Unfortunately the autocompletion isn't lightning fast.

Signed-off-by: Martin Kutlak <mkutlak@redhat.com>

### CODE REVIEW

**Comment by ernestask** [comment]
Well, sure, since it has to spawn a Python interpreter. You can write them by hand, if you want. :p

Also won’t work on zsh (and possibly friends) without flipping some switch somewhere, but that’s a different problem.

File: src/bin/Makefile.am [hunk 1]
```diff
@@ -1,3 +1,7 @@
 bin_SCRIPTS = faf faf-migrate-db
 
+bashcompdir = $(sysconfdir)/bash_completion.d
+dist_bashcomp_DATA = faf.bash_completion \
```
Comment (ernestask) [inline src/bin/Makefile.am]:
```suggestion
dist_bashcomp_DATA = \
    faf.bash_completion \
```

Comment (ernestask) [reply src/bin/Makefile.am]: Otherwise it’s a rather annoying amount of whitespace.

**Comment by ernestask** [comment]
One of our highly-trained engineers will look into that.

Thank you for using ABRT, @mastyk, we strive to deliver the best solutions for Beaker.

**Comment by mgrabovsky** [comment]
Looks fine to me.

**Pull request merged**